src/dmd/astbase.d

```diff
@@ -6396,11 +6333,11 @@ struct ASTBase
             SCstring(STC.system, TOK.at, "@system"),
             SCstring(STC.disable, TOK.at, "@disable"),
             SCstring(STC.future, TOK.at, "@__future"),
-            SCstring(0, TOK.reserved)
+            SCstring(STC.undefined_, TOK.reserved)
         ];
         for (int i = 0; table[i].stc; i++)
         {
-            StorageClass tbl = table[i].stc;
+            STC tbl = table[i].stc;
```

Comment:
Why so many changes of `StorageClass` to `STC`. `StorageClass` is a far better name. Or you can switch to `auto`/`const`.

Reply:
See my next comment.